skglm/prototype_PN/acceleration.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+from numba import float64, int32
+from numba.experimental import jitclass
+
+spec_AA = (
+    ('K', int32),
+    ('current_iter', int32),
+    ('arr_w_', float64[:, ::1]),
+    ('arr_Xw_', float64[:, ::1])
+)
+
+
+@jitclass(spec_AA)
```

Comment:
Is it clear that this class needs to be jitted ?

Reply:
Yes, it must be jitted as it is used inside a numba jitted function.
That being said, AA doesn't bring much to the solver so we won't need it.